src/test/java/io/tackle/applicationinventory/services/issues/Issue268Test.java

```diff
@@ -0,0 +1,164 @@
+package io.tackle.applicationinventory.services.issues;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.common.ResourceArg;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.InjectMock;
+import io.restassured.RestAssured;
+import io.restassured.config.EncoderConfig;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.tackle.applicationinventory.entities.ApplicationImport;
+import io.tackle.applicationinventory.entities.ImportSummary;
+import io.tackle.applicationinventory.services.BusinessServiceService;
+import io.tackle.applicationinventory.services.TagService;
+import io.tackle.commons.testcontainers.KeycloakTestResource;
+import io.tackle.commons.testcontainers.PostgreSQLDatabaseTestResource;
+import io.tackle.commons.tests.SecuredResourceTest;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.hamcrest.core.Is;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.mockito.Mockito;
+
+import javax.inject.Inject;
+import javax.transaction.HeuristicMixedException;
+import javax.transaction.HeuristicRollbackException;
+import javax.transaction.NotSupportedException;
+import javax.transaction.RollbackException;
+import javax.transaction.SystemException;
+import javax.transaction.UserTransaction;
+import javax.ws.rs.core.MediaType;
+import java.io.File;
+import java.util.Arrays;
+import java.util.Collections;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@QuarkusTest
+@QuarkusTestResource(value = PostgreSQLDatabaseTestResource.class,
+        initArgs = {
+                @ResourceArg(name = PostgreSQLDatabaseTestResource.DB_NAME, value = "application_inventory_db"),
+                @ResourceArg(name = PostgreSQLDatabaseTestResource.USER, value = "application_inventory"),
+                @ResourceArg(name = PostgreSQLDatabaseTestResource.PASSWORD, value = "application_inventory")
+        }
+)
+@QuarkusTestResource(value = KeycloakTestResource.class,
+        initArgs = {
+                @ResourceArg(name = KeycloakTestResource.IMPORT_REALM_JSON_PATH, value = "keycloak/quarkus-realm.json"),
+                @ResourceArg(name = KeycloakTestResource.REALM_NAME, value = "quarkus")
+        }
+)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class Issue268Test extends SecuredResourceTest {
```

Comment:
Better keep the naming consistent with the other issue-related test class [IssueTACKLE282Test](https://github.com/konveyor/tackle-application-inventory/blob/181b59366bb2515eaf46403dd7981111ba07615b/src/test/java/io/tackle/applicationinventory/resources/issues/IssueTACKLE282Test.java#L51) and add the link to the issue for a quick reference in case of need.
```suggestion
// https://issues.redhat.com/browse/TACKLE-268
public class IssueTACKLE268Test extends SecuredResourceTest {
```